kiwi/bootloader/config/grub2.py

```diff
@@ -657,6 +659,14 @@ def _setup_default_grub(self):
             r'(^root=[^\s]+)|( root=[^\s]+)', '', self.cmdline
         ).strip()
         if self.persistency_type == 'by-label':
+            label = re.search(r'(^root=[^\s]+)|( root=[^\s]+)', self.cmdline)
+            if label and label.groups()[-1]:
+                label_definition = label.groups()[-1].strip()
+                if 'LABEL' in label_definition:
+                    grub_default_entries['GRUB_CMDLINE_LINUX'] = \
+                        '"{0}"'.format(label_definition)
+                    grub_default_entries['SUSE_REMOVE_LINUX_ROOT_PARAM'] = \
```

Comment:
I assume adding SUSE specific value does not harm (os is just ignored by grub tooling) in case of other distributions ?

Reply:
Yes, these setting are used in the various scripts that the grub package of a distro delivers in `/etc/grub.d/` and other distros would not have any code looking for `SUSE_*` settings